SettingsList.py

```diff
@@ -3599,6 +3601,15 @@ class SettingInfos:
 
             The Poe collector will tell the reward for selling
             him Big Poes.
+
+            The sign in Deku Theater will tell the reward for showing
+            the Skull Mask.
+
+            The sign in Deku Theater will tell the reward for showing
+            the Mask of Truth.
+
+            If both Skull Mask and Mask of Truth are selected, the sign
+            in Deku Theater will tell the reward for both items.
```

Comment:
I feel like this can be reworded and combined into one paragraph. This tooltop is already verbose enough.